spec/features/form/page_routing_spec.rb

```diff
@@ -72,4 +72,28 @@
       expect(page).to have_current_path("/logs/#{id}/property-wheelchair-accessible")
     end
   end
+
+  context "routing based on start date period", js: true do
```

Comment:
I think this functionality is well tested here already https://github.com/communitiesuk/mhclg-data-collection-beta/blob/main/spec/requests/form_controller_spec.rb#L55 so these feature ones might not be necessary?

Reply:
This was originally written prior to those tests I'm happy to remove since I think this is also covered by some request specs